helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/ClusterAccessor.java

```diff
@@ -267,6 +268,129 @@ public Response getClusterConfig(@PathParam("clusterId") String clusterId) {
     return JSONRepresentation(config.getRecord());
   }
 
+
+  @PUT
+  @Path("{clusterId}/customizedstateaggregationconfig")
```

Comment:
This endpoint name is so difficult to read/write. Can you change it to snake-case: `customized-state-xxx`?

Reply:
Fixed.